.tests/config/pep/samples.csv

```diff
@@ -1,3 +1,5 @@
-sample_name,fq1,fq2,date,is_amplicon_data,technology
-illumina_shotgun_sample,data/B117.1.fastq.gz,data/B117.2.fastq.gz,2021-01-01,0,illumina
-ont_amplicon_sample,data/ont_reads.fastq.gz,,2021-01-01,1,ont
+sample_name,fq1,fq2,date,is_amplicon_data,technology,include_in_high_genome_summary
+a,data/B117.1.fastq.gz,data/B117.2.fastq.gz,2021-01-01,0,illumina,1
+No-RKI-a1,data/B117.1.fastq.gz,data/B117.2.fastq.gz,2021-01-01,0,illumina,0
+illumina_shotgun_sample,data/B117.1.fastq.gz,data/B117.2.fastq.gz,2021-01-01,0,illumina,1
+ont_amplicon_sample,data/ont_reads.fastq.gz,,2021-01-01,1,ont,1
```

Comment:
We don't have a .tests sample sheet anymore -> this is now created directly in the CI/CD. A simple merge from the master should solve this.